backend/app/library/logger/logger.go

```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	"log"
+
+	"go.uber.org/zap"
+)
+
+var logger *zap.Logger
+
+func init() {
+	var err error
+
+	logger, err = zap.NewDevelopment()
+	if err != nil {
+		log.Fatal("logger creation error")
+	}
+
+	// convert global
+	zap.ReplaceGlobals(logger)
+}
+
+func Debug(msg string, keysAndValues ...interface{}) {
+	defer logger.Sync()
+	zap.S().Debugw(msg, keysAndValues...)
+}
+
+func Info(msg string, keysAndValues ...interface{}) {
+	defer logger.Sync()
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `logger.Sync` is not checked (errcheck)